modules/cover-image-impl/src/main/java/org/opencastproject/coverimage/impl/AbstractCoverImageService.java

```diff
@@ -233,32 +220,32 @@ protected static void transformSvg(Result svg, InputSource xmlSource, Document x
       throw new IllegalArgumentException("Neither svg nor xmlSource nor xslDoc must be null");
     }
 
-    Transformer transformer;
+    Thread thread = Thread.currentThread();
+    ClassLoader loader = thread.getContextClassLoader();
+    thread.setContextClassLoader(AbstractCoverImageService.class.getClassLoader());
     try {
+      Transformer transformer;
       // CHECKSTYLE:OFF
       // Xalan Transformer can't be configured safely
       // xslDoc is an already parsed Document
-      transformer = TransformerFactory.newInstance()
-              .newTransformer(new DOMSource(xslDoc));
+      transformer = TransformerFactory
```

Comment:
This needs to use `SafeTransformerFactory` in the `common` module

Reply:
That won't work, you have to call java functions during the XSL processing.

Reply:
I tried to replace it with the SafeTransformerFactory version, but, as stated before, it won't work, and that's the reason why there is a comment above that line. Someone else tried that too.